gems/laa_multi_step_forms/app/controllers/steps/base_step_controller.rb

```diff
@@ -18,7 +18,8 @@ def update
     end
 
     def reload
-      raise 'implement this action, if needed, in subclasses'
+      @form_object.valid?
+      render :edit
     end
     # :nocov:
```

Comment:
should the :nocov: be moved up to line 19 given that it is now actual functionality